pkgs/development/ocaml-modules/chrome-trace/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildDunePackage, dune_3, dune-private-libs }:
+
+buildDunePackage rec {
+  pname = "chrome-trace";
+  inherit (dune_3) src version;
+
+  duneVersion = "3";
+
+  dontAddPrefix = true;
+
+  preBuild = ''
+    rm -r vendor/csexp
+  '';
+
+  meta = with lib; {
+    description = "Chrome trace event generation library";
+    inherit (dune_3.meta) homepage;
+    maintainers = with lib.maintainers; [ ];
```

Comment:
What’s the purpose of this line?

Reply:
it's copy-pasta from dune-site, let me remove it